qa/full-cluster-restart/src/test/java/org/elasticsearch/upgrades/FullClusterRestartIT.java

```diff
@@ -119,6 +120,10 @@ public void testSearch() throws Exception {
             client().performRequest("POST", "/" + index + "/_refresh");
         }
         assertBasicSearchWorks(index);
+        assertAllSearchWorks(index);
+        assertBasicAggregationWorks(index);
+        assertRealtimeGetWorks(index);
+        assertNewReplicasWork(index);
```

Comment:
I just like the idea of keeping the complexity around searching in the search test. That way the test for new replicas doesn't need fancy documents. It doesn't *have* fancy documents so we can be sure fancy documents have nothing to do with the test.